### DISCUSSION TITLE
CustomTkinter

### DISCUSSION THREAD

**Discussion by sumeshir26** [sumeshir26/TimerX#80] [Ideas]
Have you guys ever heard pf [CustomTkinter](https://github.com/TomSchimansky/CustomTkinter)?
It has many nice features like
- Automatic System Theme Detection
- Light and Dark Modes
- Customizable Accent Color and Corner Radius

**Comment by not-nef** [comment]
I saw that, but i dont like the theming. The sun-valley ttk theme is way better

**Comment by lg-nineoneninethreenine** [comment]
I have, but isn't it hard to add the theming in that?

**Comment by lg-nineoneninethreenine** [comment]
I even tried it, but it failed.

**Comment by sumeshir26** [comment]
what do you mean by themeng?

**Reply by lg-nineoneninethreenine** [reply]
The UI itself.
